install/kubernetes/helm/istio/charts/pilot/values.yaml

```diff
@@ -12,8 +12,10 @@ rollingMaxUnavailable: 25%
 image: pilot
 sidecar: true
 traceSampling: 1.0
-# if protocol sniffing is enabled. Default to true.
-enableProtocolSniffing: true
+# if protocol sniffing is enabled for outbound
+enableProtocolSniffingForOutbound: true
+# if protocol sniffing is enabled for inbound
+enableProtocolSniffingForInbound: false
```

Comment:
why is this being turned off?

Reply:
There is a bug when destination rule is applied. The inbound cannot inspect ALPN when TLS is enabled.